src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/GtfsRealtimeProviderImpl.java

```diff
@@ -139,12 +103,6 @@ private static float getDirVal(String direction) {
         }
     }
 
-    @Inject

```

Comment:
Same here - I think this `@Inject method` and `_gtfsRealtimeProvider` may be needed for Guice.

Reply:
This one wasn't commented out, though, right?

Reply:
Added the inject methods that I removed.